### ISSUE TITLE
Serving http.FileServer via pkger.Dir does not work on Windows?

### ISSUE THREAD

**Issue by derphilipp** [markbates/pkger#75]
_TLDR: stored index.html is not found on Windows when delievering via http.FileServer_

I packaged an `index.html` file:

```bash
pkger list 
                                                                                                                                                               
github.com/derphilipp/foo
 > github.com/derphilipp/foo:/html
 > github.com/derphilipp/foo:/html/index.html
```
And serving it works fine too:
```go
 func startPackrWebserver() {
     logrus.Info("Serving website files from packr")

     fs := http.FileServer(pkger.Dir("/html"))
     http.Handle("/", fs)
 }
```
This works for `linux/amd64`, `linux/armv7`, `darwin/amd64`

But when I crosscompile it to `windows/amd64`...

`GOOS=windows GOARCH=amd64       CGO_ENABLED=0 go build -ldflags "-extldflags '-static' -X github.com/derphilipp/foo/version.Version=${VERSION} -X github.com/derphilipp/foo/version.GitCommit=${GIT_COMMIT}${GIT_DIRTY} -X
 github.com/derphilipp/foo/version.BuildDate=${BUILD_DATE}" -o bin/windows-amd64`

...the resulting binary start the webserver but always delievers a 404 error - it seems it cannot find the index.html

The build settings are used for all the other builds too and only differ in `GOOS` and `GOARCH`.

**Comment by derphilipp** [comment]
I found out the issue, it had nothing to do with pkger, just a small bug in the build pipeline.
